vere/raft.c

```diff
@@ -1464,19 +1466,21 @@ _raft_sure(u3_noun ovo, u3_noun vir, u3_noun cor)
       u3z(ovo);
 
       // push a new event into queue
-      u3A->roe = u3nc(u3nc(vir, u3_nul), u3A->roe);
+      ret = u3nc(vir, u3_nul);
```

Comment:
It looks like we're replacing the event ovum with `u3_nul` if the state didn't change in computing the event. I don't think that's being checked for anywhere else though. We can skip persistence in that case, can we not?

Reply:
I had misremembered (or was thinking of an earlier state of these changes) -- we do check for the null ovum. But we don't apply effects if the ovum is null. I'm looking closer

Reply:
This was indeed the case, I've fixed it.

Reply:
I don't think it's a good idea to do a full equality check after each event, but that's out of scope for this. Making sure we apply the effects is good.